gromacs/fileformats/xvg.py

```diff
@@ -612,7 +613,7 @@ def plot(self, **kwargs):
 
         color = kwargs.pop('color', self.default_color_cycle)
         try:
-            cmap = matplotlib.cm.get_cmap(color)
+            cmap = matplotlib.cm.get_cmap(color) if(sys.version_info[0] < 3) else matplotlib.cm.register_cmap(color)
```

Comment:
I think [cm.get_cmap()](https://matplotlib.org/3.3.1/api/cm_api.html#matplotlib.cm.get_cmap ) is just supposed to accept "viridis" or "hot" or any of the other internal color map names.

What is the difference between importing mpl under Python 2 vs Python 3? I don't understand why we need to look a the Python version to make a mpl issue work.